client/galaxy/scripts/apps/admin.js

```diff
@@ -96,9 +97,15 @@ window.app = function app(options, bootstrapped) {
         },
 
         show_user_api_keys: function() {
-            var vuemount = document.createElement("div");
-            this.page.display(vuemount);
-            new Vue(UserAPIKeys).$mount(vuemount);
+            var vueMount = document.createElement("div");
+            this.page.display(vueMount);
+            new Vue(UserAPIKeys).$mount(vueMount);
```

Comment:
Can we augment the `display` caller such that it does the mounting for Vue-components internally.

Reply:
@guerler Yeah, that's on my list (I mentioned this in a previous Vue PR).  If you want it for this PR, I can go ahead and add something simple for full-page vue components.

Reply:
Cool. Its up to you.